web-client/iron-svelte-client/src/lib/login/login.svelte

```diff
@@ -37,12 +37,12 @@
 
                 toast.set({
                     type: 'error',
-                    message: event.data.backtrace != null ? event.data.backtrace() : event.data,
+                    message: typeof event.data !== 'string' ? event.data.backtrace() : event.data,
```

Comment:
some how this is not working anymore, had to do it in this "free of any type" way. (i.e eslint is not happy with implicit any here, but how is it working before but not now? )

Reply:
Just look at previous version, it was implicit any for both data and backtrace(). Maybe because eslint could not find the type, so it let the any slide.

Reply:
Before your modification data could be null and it's not the case anymore because you removed the ? in session-event.ts.
But it's ok in this way too